frontend/styles/components/static-content/team.scss

```diff
@@ -10,6 +10,11 @@
 
   .team-list {
     padding: 20px 0 0;
+
+    @media screen and (max-width: $breakpoint-foundation-small) {
```

Comment:
I prefer min-width, I know we've talked about this before and the project has a mix of both so not super important.